### PR TITLE
Optimize for smaller binaries

### PR DESCRIPTION
**Pull request by ZuoFuhong**
Cargo can be configured to automatically strip binaries. 

Before:

image

now:

image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@
 
 /target
 **/*.rs.bk
+.idea/
```
Comment (svenstaro) [inline .gitignore:10]: I'm not sure how I feel about editor-specific ignores here.

File: Cargo.toml [hunk 2]
```diff
@@ -15,6 +15,7 @@ lto = true
 opt-level = 'z'
 codegen-units = 1
 panic = 'abort'
+strip = true
```
Comment (svenstaro) [inline Cargo.toml:18]: We already strip in post but I suppose turning this on doesn't hurt.

File: .gitignore (hunk 1 above)
Comment (ZuoFuhong) [reply .gitignore:10]: Maybe you are not using the idea editor.

Comment (svenstaro) [reply .gitignore:10]: Ok, but at least section it of with something like `# Editor specific ignores below`

Comment (ZuoFuhong) [reply .gitignore:9]: Ok, i submitted

**Comment by svenstaro** [comment]
Thanks.

**Pull request merged**